docs/source/contributing/developing-core.md

```diff
@@ -143,6 +151,20 @@ git clone https://github.com/plone/volto.git
 cd volto
 ```
 
+Running the backend.
+
+```shell
+make-start-backend-docker
+```
+
+See the backend running at [http://localhost:8080](http://localhost:8080)
+
+In a new shell/terminal:
+
+```shell
+cd volto
+```
+
```

Comment:
I reserve further review of the above content, per https://github.com/plone/volto/issues/5556#issuecomment-1868144254